drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -172,6 +172,12 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     block_settings_key = 'drag-and-drop-v2'
     has_score = True
 
+    def max_score(self):
+        """
+        Return the problem's max score. Required by the grading system in the LMS.
+        """
+        return self.weight
```

Comment:
Thanks you @nasthagiri, I missed that! I will change `max_score` to always return `1`.